Project/src/MakeCall/CallCard.js

```diff
@@ -67,11 +68,11 @@ export default class CallCard extends React.Component {
             canSpotlight: this.capabilities.spotlightParticipant?.isPresent || this.capabilities.spotlightParticipant?.reason === 'FeatureNotSupported',
             canMuteOthers: this.capabilities.muteOthers?.isPresent || this.capabilities.muteOthers?.reason === 'FeatureNotSupported',
             canReact: this.capabilities.useReactions?.isPresent || this.capabilities.useReactions?.reason === 'FeatureNotSupported',
-            videoOn: this.call.isLocalVideoStarted,
```

Comment:
Nit: why remove the usage of `isLocalVideoStarted`? I'd suggest to keep this in as this is a sample and `isLocalVideoStarted` is a GA API on the SDK.